### ISSUE TITLE
More graceful handling of lat/long

### ISSUE THREAD

**Issue by duckinator** [RefugeRestrooms/refugerestrooms#30]
At the moment it shoves them in the URL, and if you go directly to (for example) http://www.refugerestrooms.org/bathrooms?search=Pittsburgh%2C+PA it redirects to http://www.refugerestrooms.org/bathrooms?lat=P&long=i&search=Pittsburgh%2C+PA -- not quite right.

I think the ideal solution would be to have the translation done on the server side each time, instead of altering the url. It should be possible to cache this information on the server without too much hassle.

Basically:
`http://www.refugerestrooms.org/bathrooms?search=Pittsburgh%2C+PA`
is far nicer than:
`http://www.refugerestrooms.org/bathrooms?utf8=%E2%9C%93&search=Pittsburgh%2C+PA&lat=40.44062479999999&long=-79.99588640000002`
